tchannel/tornado/connection.py

```diff
@@ -192,6 +195,8 @@ def on_error(future):
 
             if isinstance(exception, tornado.iostream.StreamClosedError):
                 self.close()
+            if message_future.running():
```

Comment:
This is a blank spot  for io_loop. It won't throw exceptions for those never triggered ``yield futures`` during shut down process.